prerequisites/software.rst

```diff
@@ -69,18 +70,26 @@ The following reverse proxies are supported:
 5. Elasticsearch (optional)
 ===========================
 
-.. note:: Package install will insist on installing elasticsearch, you can break dependencies during install if needed.
+For excellent search performance we use Elasticsearch. 
+While package install will insist on installing Elasticsearch, you can break those dependencies if needed.
 
 .. warning:: Please note that if you do not install and use Elasticsearch, the search will be very limited!
    We recommend using Elasticsearch, as it will boost the usuage of Zammad greatly!
 
-For excellent search performance we use Elasticsearch.
-The following Elasticsearch versions are supported:
 
-* Elasticsearch 5.5 with ``mapper-attachments`` plugin
-* Elasticsearch 5.6, 6.x & 7.x with ``ingest-attachment`` plugin
+The Elasticsearch support differs on the Elasticsearch and Zammad version you're using:
 
-.. warning:: Please note that Elasticsearch 6.x and 7.x support came with Zammad 3.1.
-   If you try to use Elasticsearch newer than 5.6.x on Zammad 3.0 and earlier, your search index will **not work**.
+.. csv-table:: Compatibility to Elasticsearch
+   :header: "Zammad version", "Supported Elasticsearch version"
+   :widths: 20, 20
 
-.. warning:: Please note that we will be dropping Elasticsearch support prior 5.5.x on future releases.
+   "3.4+", "5.5 up to 7.6.x"
+   "3.3", "2.4 up to 7.6.x"
+   "3.2", "2.4 up to 7.5.x"
+   "3.1", "2.4 up to 7.4.x"
+   "2.0 - 3.0", "2.4 up to 5.6.x"
```

Comment:
```suggestion
   "3.4+", "5.5–7.6"
   "3.3", "2.4–7.6"
   "3.2", "2.4–7.5"
   "3.1", "2.4–7.4"
   "2.0–3.0", "2.4–5.6"
```

I used [endashes](https://en.wikipedia.org/wiki/Dash#En_dash) here, which is preferred by many style guides for specifying ranges.

Also, I personally that the `.x` is implied and adds visual clutter, but I'm willing to budge on this if you think otherwise.